asdf_astropy/converters/unit/tests/test_unit.py

```diff
@@ -0,0 +1,55 @@
+
+import warnings
+
+import asdf
+from asdf.tests import helpers
+import astropy
+from astropy import units
+from packaging.version import Version
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def remove_astropy_extensions():
+    """
+    Disable the old astropy extension so that it doesn't
+    confuse our test results.
+    """
+    with asdf.config_context() as config:
+        config.remove_extension(package="astropy")
+        yield
```

Comment:
Since this gets used 3 times, you may wanat to put this in a `conftest.py`, turn off `autouse`, and make the fixture dependency explicit in each test that uses it.

Reply:
In the end this is going to be `autouse=True` for the entire package, but we can't do that until all the converters are implemented.  I thought it'd be easier to go through and delete all these fixtures vs delete the fixture name from individual test method signatures.

Reply:
Ah, I see.  Carry on.  👍